src/shared/components/case-editor/case-access-utils/index.ts

```diff
@@ -85,15 +85,15 @@ export class CaseAccessUtils {
         const payload: RoleRequestPayload = {
             roleRequest: {
                 assignerId: assignerId,
-                process: 'specific-access',
-                reference: assignerId,
-                replaceExisting: false
+                process: 'challenged-access',
```

Comment:
Does this mean specific access requests will no longer work?

Reply:
Very good point. I did not realise that same function is being used by Specific access. I will change it now

Reply:
Done